src/ruby/yast/core_ext.rb

```diff
@@ -0,0 +1 @@
+require_relative "core_ext/string"
```

Comment:
Hmmm, in the particular use case that originated this PR (just cleaning a single string) I like the suggested approach of using ```extend```. But if we think about other use cases for the core extensions idea, it's probably too cumbersome (i.e., not that convenient anymore). To prevent String to become fat and eventually dangerous I'd simply use refinements. I don't find it too much magical, you have to explicitly require the extension anyways.